uniswap_swap_bot/README.md

```diff
@@ -0,0 +1,26 @@
+# Large Tether Transfer Agent
```

Comment:
This README should be updated, as this is the boilerplate README autogenerated by the Forta SDK.